fractal_server/app/runner/v2/_local_experimental/executor.py

```diff
@@ -67,11 +66,15 @@ def _terminate_processes(self):
         Running on '_shutdown_file_thread'.
         """
         logger.info("Start terminating FractalProcessPoolExecutor processes.")
+        # We use 'psutil' in order to easily access the PIDs of the children.
         if self._processes is not None:
             for pid in self._processes.keys():
-                logger.debug(f"Sending SIGTERM to process {pid}")
-                os.kill(pid, signal.SIGTERM)
-                logger.debug(f"Process {pid} terminated.")
+                parent = psutil.Process(pid)
+                children = parent.children(recursive=True)
+                for child in children:
+                    child.kill()
+                parent.kill()
+                logger.debug(f"Process {pid} and its children terminated.")
```

Comment:
Do we see these logs anywhere?

Reply:
We do, after 3b7c5d7c6c751c0b53860bc539174b2c959ade48